docs/source/cleanlab/datalab/guide/table.rst

```diff
@@ -0,0 +1,186 @@
+.. tabs::
+
+    .. tab:: Classification task
```

Comment:
opened another TODO related to this table:
https://github.com/cleanlab/cleanlab/issues/1133

But can save for the future

Reply:
I agree